src/WebSDL/settings/settings_template.json

```diff
@@ -3,8 +3,11 @@
   "host": "-optional",
   "recaptcha_secret_key": "-optional",
   "recaptcha_user_key": "-optional",
-  "email_sender": "password.reset@data.envirodiy.org",
+  "password_email_sender": "password.reset@data.envirodiy.org",
```

Comment:
Same thing as earlier - let's be clear we don't receive emails to this address - no_reply@data.envirodiy.or